codersdk/deployment.go

```diff
@@ -103,6 +104,7 @@ var FeatureNames = []FeatureName{
 	FeatureControlSharedPorts,
 	FeatureCustomRoles,
 	FeatureMultipleOrganizations,
+	FeatureWorkspacePrebuilds,
```

Comment:
I don't think any of this should be included in this PR; it's not necessary for the claiming code to be merged - is it?

Reply:
yes